### ISSUE TITLE
GNU radio companion does not start

### ISSUE THREAD

**Issue by ottomc2** [gnuradio/gnuradio#6187]
I installed it recently and I have an icon but the app does not want to start.
a prompt window comes up, few lines appear in it, and everything disappears in few seconds.
Latest revision radioconda 2002.6.24.0
installed successfully
Typing like "gnuradio-companion &" in a prompt window does not help much.
En error message comes: cannot import gnu radio

**Comment by willcode** [comment]
@ottomc2 Radioconda is here https://github.com/ryanvolz/radioconda. If you think this is a GNU Radio problem, please post the error you are seeing.

**Comment by ottomc2** [comment]
Yes, I fetched the version available on that very page:
https://github.com/ryanvolz/radioconda
There is the only version for windows available there:
radioconda-Windows-x86_64.exe
<https://glare.now.sh/ryanvolz/radioconda/radioconda-.*-Windows-x86_64.exe>
I ran that one and got a message that GNU radio was installed successfully.
After that, I have not been able to start the application.
That's the problem I experience.
